src/material-experimental/mdc-slider/slider.scss

```diff
@@ -6,3 +6,8 @@
 .mdc-slider {
   display: block;
 }
+
+.mat-ripple {
```

Comment:
This should be qualified with another class selector (probably just nested inside `.mdc-slider`)

Reply:
Oh, I remember from our discussion that you mentioning that I should move this into a separate file slider-thumb.scss. Am I remembering that correctly?

Reply:
It should be something like `.mat-mdc-slider .mat-ripple`, because the current selector targets all ripples on the page.